### ISSUE TITLE
ComposeMultipleContentEmitters check doesn't trigger in for loops

### ISSUE THREAD

**Issue by dfcarvalho** [slackhq/compose-lints#197]
Hi :)

I noticed the following code doesn't trigger the ComposeMultipleContentEmitters warning:

```
@Composable
fun MultipleContent(otherTexts: List<String>, modifier: Modifier = Modifier) {
    Text("text 1")
    for (otherText in otherTexts) {
        Text(otherText)
    }
}
```

Or even this one:

```
@Composable
fun MultipleContent(texts: List<String>, modifier: Modifier = Modifier) {
    for (text in texts) {
        Text(text)
    }
}
```

Not sure if something can be done about it, but thought I'd report anyway.

Cheers!

**Comment by ZacSweers** [comment]
We could do a trivial check for an enclosing `for` loop or `forEach` lambda, but yeah probably can only do so much here
